src/quote/quote.service.ts

```diff
@@ -131,4 +137,93 @@ export class QuoteService implements OnModuleInit {
 
     await this.quoteRepository.save(quoteEntities);
   }
+
+  async getLatestPrice(deployment: Deployment, address: string, currencies: string[]): Promise<any> {
+    const enabledProviders = this.priceProviders[deployment.blockchainType].filter((p) => p.enabled);
+
```

Comment:
nitpicking (kinda) - there's a lot of `address.toLowerCase()`, some inside loops, inside this function - easily improved by adding here a 
`const lcAddress = address.toLowerCase()`